Replace outdated Explanation in PostAllAssetLinksByID Table (#485)
admin-shell-io/aas-specs-api: documentation/IDTA-01002-3/modules/ROOT/pages/specification/interfaces.adoc

|===
h|Operation Name 4+e|[[PostAllAssetLinksById]]PostAllAssetLinksById
h|Explanation 4+a|

> Creates or replaces all asset identifier key-value-pairs linked to an Asset Administration Shell to edit discoverable content.

Reviewer: ```suggestion
Creates or replaces all asset links associated to the Asset Administration Shell.
```

Reviewer: we should not talk about key-value-pairs because they are not. They are triples: name, vale, externalSubjectId
